test/quantization/test_workflow_module.py

```diff
@@ -1056,6 +1056,7 @@ def _test_learnable_backward_per_tensor(self, X, device, scale_base, zero_point_
     @given(X=hu.tensor(shapes=hu.array_shapes(1, 5,),
                        elements=hu.floats(-1e3, 1e3, allow_nan=False, allow_infinity=False),
                        qparams=hu.qparams(dtypes=torch.quint8)))
+    @unittest.skip("temporarily disable the test to see if test_shape_ops is passing or failing?")
```

Comment:
What is the different between the before PR and the after PR in what nonzero computes in these tests? It may be helpful to add a debug print to nonzero in the current nightly, then apply your changes with the same debug print.